joomla/joomla-cms
# [4.0] Calendar translation without extra js files

Alternative for  #34819 .

### Summary of Changes

The translation done with use of existing language constants.
I have removed all translations files in `system/js/fields/calendar-locales/`

Added 2 news strings: `JLIB_HTML_BEHAVIOR_AM`, `JLIB_HTML_BEHAVIOR_PM`

~~New method `Language::getLocalNumbers()`, this need for `fa-IR`, `pr-AF` languages that use `jalali` calendar.~~
~~Please someone warn maintainers of these languages to add ` <numbers>۰,۱,۲,۳,۴,۵,۶,۷,۸,۹</numbers>` in their metadata.~~

I think it is better than maintain 100 JS files.


### Testing Instructions

Apply path, run `npm install`.
Make sure calendar works.

Please test `jalali` calendar also. For this can use Persian lang pack from 3.9



### Actual result BEFORE applying this Pull Request
works


### Expected result AFTER applying this Pull Request
works without `system/js/fields/calendar-locales/`


### Documentation Changes Required

~~A new `Language::getLocalNumbers()` method read the numbers from language metadata.~~
~~For `jalali` calendar the language XML should have a metadata `numbers`, eg: `<numbers>۰,۱,۲,۳,۴,۵,۶,۷,۸,۹</numbers>`~~
